### PR TITLE
Dump choices and created vote values

### LINKED ISSUE #51: Add the burn remainder amount and transaction hash to the funding round table
Request from Alex Napheys:

- [ ] add the burn remainder amount and the transaction hash column to the funding round table
- [ ] update the DAOBot to fill in the burn remainder field once the round is finished and the burn amount is calculated
The transaction hash will be entered manually @daneklustig

### PR DESCRIPTION
**Pull request by KatunaNorbert**
Fixes #51.

### CODE REVIEW

**Comment by KatunaNorbert** [comment]
To do:

- [ ] Display choices inside GSheet
- [ ] Display vote created timestamp 
- [ ] Make sure `syncGSheetsActiveProposalVotes()` works for both voting strategies
- [ ] use `syncGSheetsActiveProposalVotes()` to generate vote details, more specifically(`dumpFromSnapshotRawToGSheet()`) for rounds 5-11

File: src/airtable/process_airtable_funding_round_complete.js [hunk 1]
```diff
@@ -116,8 +116,8 @@ const processFundingRoundComplete = async (curRound, curRoundNumber) => {
   // Get the sheet, otherwise create it
   let sheet = await getValues(oAuth, sheetName, 'A1:B3')
   if (sheet === undefined) {
-    sheet = await addSheet(oAuth, sheetName)
-    Logger.log('Created new sheet [%s].', sheetName)
+    sheet = await addSheet(oAuth, sheetName, curRoundNumber)
+    console.log('Created new sheet [%s].', sheetName)
```
Comment (trizin) [inline src/airtable/process_airtable_funding_round_complete.js:120]: Please use `Logger.log`

File: src/gsheets/sync_gsheets_active_proposal_votes_snapshot.js [hunk 2]
```diff
@@ -285,9 +300,10 @@ const dumpRoundSummaryToGSheets = async (
   // Get the sheet, otherwise create it
   const sheetName = `Round ${curRoundNumber} Results`
   var sheet = await getValues(oAuth, sheetName, 'A1:B3')
+  console.log(sheet)
```
Comment (trizin) [inline src/gsheets/sync_gsheets_active_proposal_votes_snapshot.js:303]: I think this `console.log` is for debugging.

File: src/gsheets/sync_gsheets_active_proposal_votes_snapshot.js [hunk 3]
```diff
@@ -285,9 +300,10 @@ const dumpRoundSummaryToGSheets = async (
...
   if (sheet === undefined) {
-    await addSheet(oAuth, sheetName)
-    Logger.log('Created new sheet [%s] at index 0.', sheetName)
+    await addSheet(oAuth, sheetName, curRoundNumber)
+    await console.log('Created new sheet [%s] at index 0.', sheetName)
```
Comment (trizin) [inline src/gsheets/sync_gsheets_active_proposal_votes_snapshot.js:306]: Please use `Logger.log`

File: src/test/gsheets/test_gsheets_sync.test.js [hunk 4]
```diff
@@ -7,11 +7,21 @@ dotenv.config()
 const {
   syncGSheetsActiveProposalVotes
 } = require('../../gsheets/sync_gsheets_active_proposal_votes_snapshot')
+const { BallotType } = require('../../snapshot/snapshot_utils')
+
+beforeEach(() => {
+  jest.setTimeout(20000)
```
Comment (trizin) [inline src/test/gsheets/test_gsheets_sync.test.js:13]: The timeout is `30000` in `package.json`

**Review by trizin — COMMENTED** [review]
Great work! For consistency, it is better to use `Logger.log`.

**Review by trizin — APPROVED** [review]

**Pull request merged**